python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -2333,14 +2333,16 @@ def convert_layer_norm(node, **kwargs):
     axes = int(attrs.get('axis', -1))
     eps = attrs.get('eps', 9.99999975e-06)
 
+    input_type = int(kwargs['in_type'])
+    dtype = onnx.mapping.TENSOR_TYPE_TO_NP_TYPE[input_type]
 
     nodes = [
         create_tensor([axes], name+"_axes", kwargs["initializer"]),
         create_tensor([axes+1], name+"_axes+1", kwargs["initializer"]),
         create_tensor([], name+"_void", kwargs["initializer"]),
         create_const_scalar_node(name+'_0_s', np.int64(0), kwargs),
         create_const_scalar_node(name+'_1_s', np.int64(1), kwargs),
-        create_const_scalar_node(name+"_2_s", np.int64(2), kwargs),
+        create_const_scalar_node(name+"_2_s", np.array(2, dtype=dtype), kwargs),
```

Comment:
Curious, why do we use array for this const node?

Reply:
This is a constant node. We just need to control the dtype so now we cannot use np.int64 or for that same reason np.float32 any more

Reply:
Makes sense, thanks!